.travis.yml

```diff
@@ -1,11 +1,35 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "0.12.7"
-  - "stable"
-services: mongodb
+  - 0.12.18
```

Comment:
just noting that this is actually a version drop ... from `0.12.7` to `0.12.18` but is consistent with the other services